### ISSUE TITLE
tsh: Confusing PuTTY key generation error message

### ISSUE THREAD

**Issue by zmb3** [gravitational/teleport#23778]
When debug logging is enabled for `tsh` on Windows, the following message is often included:

```
2023-03-29T18:36:08+05:30 ←[36mINFO←[0m [CLIENT]    Waiting for response at: http://127.0.0.1:60827. client\redirect.go:157
If browser window does not open automatically, open it by clicking on the link:
 http://127.0.0.1:60827/4c448696-2d8b-4b8d-bb0f-ab615bb278d5
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [CLIENT]    Got response from browser. client\weblogin.go:377
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [KEYSTORE]  Teleport TLS certificate valid until "2023-03-29 23:00:20 +0000 UTC". client\client_store.go:91
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [KEYAGENT]  Deleting obsolete stored key with index {ProxyHost:teleport.example.com Username:foo@example.com ClusterName:teleport.example.com}. client\keyagent.go:518
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [KEYSTORE]  Cannot convert private key to PPK-formatted keypair. error:[<nil>] client\keystore.go:173
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [KEYSTORE]  Adding known host teleport.example.com with proxy teleport.example.com client\trusted_certs_store.go:393
2023-03-29T18:36:54+05:30 ←[36mINFO←[0m [KEYAGENT]  Loading SSH key for user "foo@example.com" and cluster "teleport.example.com". client\keyagent.go:195
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [KEYSTORE]  Teleport TLS certificate valid until "2023-03-29 23:06:53 +0000 UTC". client\client_store.go:91
2023-03-29T18:36:54+05:30 ←[37mDEBU←[0m [KEYSTORE]  Teleport TLS certificate valid until "2023-03-29 23:06:53 +0000 UTC". client\client_store.go:91
```

It reads like there was an error, but then says that error is nil. This causes confusion.

Clean up the message and make it clear what (if anything) went wrong.

**Comment by webvictim** [comment]
Can I get the full error log including surrounding context? It's difficult to tell why this is actually happening, and I'd rather fix the underlying issue than just remove the debug print.

**Comment by webvictim** [comment]
Fixed by #23899
